### DISCUSSION TITLE
[Feature Request] - Total percent and time complete

### DISCUSSION THREAD

**Discussion by ghost** [PaulWoitaschek/Voice#1422] [Ideas]
Would it be possible to add the book percentage progress to the screen?
Also, perhaps a total time completed.
Thanks,

App-Version: 3.6.1
Phone Name: Google Pixel 2 XL
Android Version: 8.1

**Comment by ghost** [comment]
@PaulWoitaschek @0rAX0 Please consider adding this to the lockscreen notification as well.

**Comment by rollecode** [comment]
I vote for this feature. Would be great as a GoodReads user (to update progress). Currently I don't know how much have I already listened... I'm not an app designer so I can just hope this to be implemented. Thanks for the great app so far!

**Comment by rollecode** [comment]
Just to clarify there already is progress, but not percent. "Show percent" could be a setting so it would be able to switch in place of the total time in the index section, so it's not really a design thing then.

**Comment by stevecopley** [comment]
I can't seem to see any sort of overall progress bar on the main screen, neither time nor percentage, so I too would like to give this request a boost. On the notification I see overall progress, but on the main screen only the chapter progress, unless I'm missing something...

**Reply by Ysa423** [reply]
I would love to have this feature besides the remaining time to have the % of of the progress. I would agree it is sufficient to have an overall % not for ever chapter.

**Comment by Ysa423** [comment]
I would love to have this feature besides the remaining time to have the % of of the overall progress.

**Comment by techiesurath** [comment]
I also vote for this feature request. As an avid audio-book listener I am always curious about how much progress I have made into a book.

**Comment by Amphobet** [comment]
Since this has been a request for 5 years, I'm not holding out much hope, but I also request this.

**Comment by apederson94** [comment]
I just added a basic implementation for this! A nice evening project to give us all a nice QoL update! #2334

**Reply by pedrojvaz** [reply]
Still not visible?

**Reply by apederson94** [reply]
> Still not visible?

That's because @PaulWoitaschek hasn't reviewed this MR or the other one related to percentage yet. I'm not sure how to get MRs reviewed so no idea how to get this code into the app 🤷🏻

**Reply by pedrojvaz** [reply]
I don't understand what I see but it seems that the branch #2334 has conflicts?

![image](https://github.com/user-attachments/assets/1a2a6dbf-f2ca-4431-b5b9-0e711a34db42)

**Comment by areksobiczewski** [comment]
I really need that future!

Usually audiobooks are divided into chapters (think of .m4b files), but I have some that are multihour, one mp3 file. Navigating between an audiobook of that kind and a paperback / ebook, if needed is a serious PITA. The most convenient way would be to know the exact % postition in a long audio file. This way it's ease to find out the actual posiotion between different media.

I can't wait for the above changes by **apdederson94** to be integrated into the app.

**Comment by Ysa423** [comment]
Hi Paul :) I use Android (Samsung Galaxy S24) an I don't see any
percentage. Maybe I dont know how to look😅 But in the overview list it's
not to be seen or in the notification bar either when its playing. Just to
be sure I have the application up to date 😊 If I am not checking it well
let me know 😊 and thank you for your efforts.

**Comment by dessalines** [comment]
Also, a remaining time would be very useful.
